main.go

```diff
@@ -18,22 +18,38 @@ limitations under the License.
 package main
 
 import (
-	log "github.com/sirupsen/logrus"
+	"fmt"
 	"os"
+	"strconv"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	aggregator "github.com/nomad-node-problem-detector/aggregator"
 	config "github.com/nomad-node-problem-detector/config"
 	detector "github.com/nomad-node-problem-detector/detector"
 	"github.com/urfave/cli/v2"
 )
 
+var (
+	Timestamp string
+	GitCommit string
+)
+
 func main() {
+	// convert the timestamp to a date time to populate correctly the
+	// information for the app
+	i, err := strconv.ParseInt(Timestamp, 10, 64)
+	if err != nil {
+		panic(fmt.Sprintf("failed to convert %s to an integer: %v", Timestamp, err))
+	}
+	compiledAt := time.Unix(i, 0)
+
 	app := &cli.App{
 		Name:                 "npd",
 		Usage:                "Nomad node problem detector",
-		Version:              "v1.0.0",
-		Compiled:             time.Now(),
+		Version:              GitCommit,
+		Compiled:             compiledAt,
```

Comment:
Suggestion:

```
Compiled: time.Unix(i, 0),
```
If you feel defining explicit variable `compiledAt` is more readable, feel free to ignore.